examples/measure.js

```diff
@@ -138,6 +132,10 @@ var map = new ol.Map({
 
 map.on('pointermove', pointerMoveHandler);
 
+map.on('mouseout', function() {
```

Comment:
Indeed. I wanted to observe `mouseout` on `$(map.getViewport())` actually.
Good catch.